graph_def_editor/node.py

```diff
@@ -779,16 +790,44 @@ def _decode_inputs(inputs, # type: Iterable[str]
   #   "^node_name" --> Control input from indicated node
   #   "node_name" --> Input from output number 0 of indicated node
   #   "node_name:ix" --> Input from output number <ix> of indicated node
+  #   "node_name:arg_def_name:ix" --> Input from arg_def's output number <ix> of indicated node
   # Start by filtering out the control inputs and turning "node_name" into
   # "node_name:0".
   input_names = [_canonicalize_output_name(n) for n in inputs
                  if not n.startswith("^")]
   input_tensors = []
   for name in input_names:
     # Name is in form "node:output number"
-    node_name, output_ix_name = name.split(":")
-    output_ix = int(output_ix_name)
-    input_tensors.append(g[node_name].output(output_ix))
+    parts = name.split(":")
+    if len(parts) == 2:
+      node_name, output_ix_name = name.split(":")
+      output_ix = int(output_ix_name)
+      input_tensors.append(g[node_name].output(output_ix))
+    elif len(parts) == 3:
+      # FuncGraph is using different format for input tensor definitions.
```

Comment:
Yikes! Glad I didn't have to write this part :)

Reply:
:-)

Reply:
Thanks for the thorough review!!!